spec/src/main/asciidoc/fallback.asciidoc

```diff
@@ -94,5 +94,28 @@ When `fallbackMethod` is used a `FaultToleranceDefinitionException` will be thro
 The parameter `value` and `fallbackMethod` on `@Fallback` cannot be specified at the same time.
 Otherwise, the `FaultToleranceDefinitionException` exception will be thrown.
 
-The fallback should be triggered when an exception occurs.
-For instance, `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException` should trigger the fallback.
+The fallback might be triggered when an exception occurs (e.g. `BulkheadException`, `CircuitBreakerOpenException`, `TimeoutException`, etc), detailed below.
+When a method returns and the Fallback policy is present, the following rules are applied:
+
+* If the method returns normally (doesn't throw), the result will be simply returned.
+* Otherwise, if the thrown object is assignable to any value in the `skipOn` parameter, the thrown object will be rethrown.
+* Otherwise, if the thrown object is assignable to any value in the `applyOn` parameter, the specified fallback will be triggered.
+* Otherwise the thrown object will be rethrown.
+In the following example, all exceptions assignable to `ExceptionA` and `ExceptionB`, except the ones assignable to `ExceptionBSub` will trigger the fallback operation.
+[source, java]
+----
+
+    @Retry(maxRetries = 2)
+    @Fallback(applyOn={ExceptionA.class, ExceptionB.class}, skipOn=ExceptionBSub.class, fallbackMethod= "fallbackForServiceB")
```

Comment:
Deleted the line. As for demonstrating the criteria, I prefer to specify the custom exception instead of relying developers knowing the hierarchy of FileNotFoundException is a subclass of IOException. For usage wise, the exceptions will be mostly user defined exception. I think the sentence above the code snippet explained the intent.